README.md

```diff
@@ -866,6 +866,54 @@ To access the provider, get it from the component injector using the `fromCompon
 spectator.get(FooService, true)
 ```
 
+## Directive Providers
```

Comment:
I don't think we need to bloat the README with this example. You can add one sentence in the previous section:

The same rules also apply to directives using the `directiveProviders` and `directiveMocks`.